packages/webpack/src/runtime/runtimeUnlocked.js

```diff
@@ -0,0 +1,47 @@
+/// <reference path="./lavamoat.d.ts" />
+/* global LAVAMOAT */
+
+const { create, freeze, defineProperty } = Object
+const warn = typeof console === 'object' ? console.warn : () => {}
```

Comment:
what are we targeting that does not define `console`?

Reply:
hostile environments of someone else's websites